changelogs/fragments/6384-gitlab-refactor.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - gitlab module utils - refactor function in from modules (https://github.com/ansible-collections/community.general/pull/6384).
```

Comment:
I would probably leave this one away, and just mention changes for the modules.

Reply:
done!